app/src/main/kotlin/org/cru/godtools/ui/languages/LanguageName.kt

```diff
@@ -41,24 +41,34 @@ internal fun LanguageName(language: Language, modifier: Modifier = Modifier) {
 
     LanguageName(
         displayName = remember(context, language, appLanguage) { language.getDisplayName(context, appLanguage) },
-        nativeName = remember(context, language) { language.getDisplayName(context, language.code) },
+        nativeName = remember(context, language) {
+            if (language.isForcedName) {
+                null
+            } else {
+                language.getDisplayName(context, language.code)
+            }
+        },
         modifier = modifier,
     )
 }
 
 private const val LANGUAGE_NAME_GAP = "[gap]"
 
 @Composable
-private fun LanguageName(displayName: String, nativeName: String, modifier: Modifier = Modifier) {
+private fun LanguageName(displayName: String, modifier: Modifier = Modifier, nativeName: String? = null) {
     val color = LocalTextStyle.current.color.takeOrElse { LocalContentColor.current }
     val displayNameColor = color.let { it.copy(alpha = it.alpha * 0.60f) }
 
     Text(
         remember(displayName, nativeName, color, displayNameColor) {
+            val name = nativeName ?: displayName
+            val secondName = if (nativeName.isNullOrBlank()) null else displayName
```

Comment:
this still because more direct if you move the `name` and `secondName` vals to be the actual parameters and just call it like:

```kotlin
    LanguageName(
        name = remember(context, language) { language.getDisplayName(context, language.code) },
        secondName = remember(context, language, appLanguage) { language.takeUnless { it.isForcedName }?.getDisplayName(context, appLanguage) },
        modifier = modifier,
    )
```